web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1450,11 +1489,10 @@ namespace com.keyman.keyboards {
      * @param       {Object}      Pk      Keyboard  object
      * Description  Register and load the keyboard
      */
-    _registerKeyboard(Pk) {
-      // If initialization not yet complete, list the keyboard to be registered on completion of initialization
+    async _registerKeyboard(Pk) {
+      // Ensure keymanweb is initialized before continuing to register keyboards
       if(!this.keymanweb.initialized) {
-        this.deferredKR.push(Pk);
-        return;
+        let result = await this.deferment;
```

Comment:
Is dropping the assignment not possible?  `result` immediately goes out of scope, after all.

```suggestion
        await this.deferment;
```

Reply:
Reminder:  this was given a :+1:, but has not yet been addressed or found non-viable.

Reply:
ah, I lost track of this in all the comments. 👍 for real now.